.github/workflows/dev-pr.yml

```diff
@@ -0,0 +1,59 @@
+name: Dev PR
+on:
+  pull_request_review:
+    types:
+      - submitted
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - edited
+      - labeled
+      - unlabeled
+      - synchronize
+      - ready_for_review
+      - unlocked
+    branches:
+      - dev
+      - main
+
+jobs:
+  dev-pr:
+    runs-on: ubuntu-latest
+    if: github.base_ref == "dev"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ensure checklists are filled
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const script = require('.github/actions/checklist.js').check;
+            await script({context});
+      - name: Ensure title update with changes type
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const script = require('.github/actions/pr-title.js').update;
+            await script({ context, github });
+      - name: automerge
+        uses: pascalgn/automerge-action@v0.14.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          MERGE_LABELS: automerge
+          MERGE_METHOD: squash
+          MERGE_COMMIT_MESSAGE: pull-request-title
+          MERGE_REQUIRED_APPROVALS: 2
```

Comment:
I believe we settled on 1 approval for now?

Reply:
👍